cypress/integration/ut2019.spec.js

```diff
@@ -1,14 +1,55 @@
-describe('ut2019', function() {
-	const email = 'test.user@ut2019.ee';
-	const name = 'Test User';
-	const message = 'Custom message!';
-	it('Submit form successfully', function() {
-		cy.clearCookies();
-		cy.visit('https://ut2019.voog.com/contact');
-		cy.get('.contact-name input').type(name);
-		cy.get('.contact-email input').type(email);
-		cy.get('.contact-message textarea').type(message);
-		cy.get('input[type="submit"]').click();
-		cy.get('.form_notice').should('have.text', 'Thank you!');
-	});
-});
+//List of different language options
+describe('generated from a list', () => {
+    const operations = [{
+            value: "et",
```

Comment:
You were able to implement the mechanism for running the test in different language versions of the site, but we did not see any code, which would actually check that the Estonian or English version of the app was opened. You should for example test that the page title is displayed in the correct language or the form submission message has the correct translation.